src/txn/txn_ckpt.c

```diff
@@ -606,6 +609,9 @@ __checkpoint_prepare(WT_SESSION_IMPL *session, const char *cfg[])
 			    sizeof(timestamp_config),
 			    "read_timestamp=%s", timestamp_buf));
 			txn_cfg[2] = timestamp_config;
+			__wt_verbose(session, WT_VERB_TIMESTAMP,
+			    "Checkpoint requested at stable timestamp: %s",
+			    timestamp_config);
```

Comment:
@sueloverso , I thought and spent sometime trying to make __wt_verbose_timestamp more generic but it proved to be more work than the time permitted. We can work on that as and when needed. In this function here, we do not have timestamps, but have strings instead and it would be quite a lot of work to fit into how __wt_verbose_timestamp is written. I am inclined to use __wt_verbose here and change the actual message to match that output by __wt_verbose_timestamp.